srcCpp/PerftestTransport.h

```diff
@@ -10,6 +10,29 @@
 #include <map>
 #include <sstream>
 #include "ParameterManager.h"
+#include <limits.h>
+
+
+#define DEFAULT_MESSAGE_SIZE_MAX 65536
+#define MESSAGE_SIZE_MAX_NOT_SET LONG_MAX
+
+/*
+ * This const is used to calculate the maximum size that a packet can have. This
+ * number has to be substracted to the message_size_max.
+ * We calculate this as:
+ * - COMMEND_WRITER_MAX_RTPS_OVERHEAD <- Size of the overhead for RTPS in the
+ *                                       worst case
+ * - 48 <- Max transport overhead (this would be for ipv6).
+ * - Encapsulation (RTI_CDR_ENCAPSULATION_HEADER_SIZE) + aligment in the worst case (3)
+ *
+ * TODO: that Encapsulation should be taken out from the sample instead of here.
+ */
+
+#ifndef RTI_MICRO
+  #define MESSAGE_OVERHEAD_BYTES (COMMEND_WRITER_MAX_RTPS_OVERHEAD + 48 + RTI_CDR_ENCAPSULATION_HEADER_SIZE + 3)
+#else
+  #define MESSAGE_OVERHEAD_BYTES 700
```

Comment:
Why this is 700?

Reply:
This value needs to be the same as above but for micro. This requires a fix.